### ISSUE TITLE
Upgrade from 4.32.2 to 4.33.1 to 5.0.3 removes ServiceControl.Audit/ServiceControlQueueAddress from config

### ISSUE THREAD

**Issue by Floopy-Doo** [Particular/ServiceControl#3905]
#### Description
When upgrading ServiceControl audit instance from 4.33.2 to 4.33.1 to 5.0.3, the new instance removes the config value set for the key "ServiceControl.Audit/ServiceControlQueueAddress". This causes the following error https://docs.particular.net/servicecontrol/troubleshooting#no-destination-specified-for-message. 

#### Expected behavior
Key ServiceControl.Audit/ServiceControlQueueAddress is not deleted during the migration

#### Actual behavior
Key ServiceControl.Audit/ServiceControlQueueAddress is deleted during the migration

#### Versions
Upgrade path: 4.32.2 -> 4.33.1 -> 5.0.3

Please list the version of the relevant packages or applications in which the bug exists.        

### Steps to reproduce

Create new instance with Version 4.32.2 ServiceControl Management. 
Ensure Key "ServiceControl.Audit/ServiceControlQueueAddress" is set 
Migrate to 4.33.1 with ServiceControl Management
Migrate to 5.0.3 with ServiceControl Management

#### Workarounds
Manually set key "ServiceControl.Audit/ServiceControlQueueAddress" in config to correct queue.

#### Possible solutions
* Not delete config key
* or: Highlight that config need to be manually migrated

**Comment by SzymonPobiega** [comment]
Hi!

Thanks for reporting it. We tried to reproduce the issue (We used the sequence of versions from the title: 4.32.2 to 4.33.1 to 5.0.3. It looks like the one in the description has a typo because there is no 4.33.2 version) but the key has not been removed and SC appears to be working as expected. Would you be able to check it on a different environment? Or provide us with the exact stack trace of the exception you see when the key is missing?

**Comment by Floopy-Doo** [comment]
You are correct, I made a typo in the description.
During the migration we had no exceptions. The exception showed up in the logs after we started the freshly migrated audit instance. (the stack trace is the same as provided the the documentation -> https://docs.particular.net/servicecontrol/troubleshooting#no-destination-specified-for-message)

We will have an upcoming migration on our production environment in January. I will keep close eye on the changes to the ServiceControl.exe.config file during the migration. I should also be able to track down the exact version.

I will keep you posted on the updates.
